lib/mws/sellers/client.rb

```diff
@@ -37,6 +37,15 @@ def get_service_status
         operation('GetServiceStatus')
         run
       end
+
+      # Gets the MWS Auth Token of the seller account
+      #
+      # @see http://docs.developer.amazonservices.com/en_US/auth_token/AuthToken_GetAuthToken.html
+      # @return [Peddler::XMLParser]
+      def get_auth_token
+        operation('GetAuthToken')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.